app/src/androidTest/java/org/mozilla/fenix/ui/robots/HistoryRobot.kt

```diff
@@ -85,16 +85,7 @@ class HistoryRobot {
     fun verifyDeleteSnackbarText(text: String) = assertSnackBarText(text)
 
     class Transition {
-        fun closeMenu(interact: HistoryRobot.() -> Unit): Transition {
-            closeButton().click()
-
-            HistoryRobot().interact()
-            return Transition()
-        }
-
         fun goBackToBrowser(interact: BrowserRobot.() -> Unit): BrowserRobot.Transition {
-            closeButton().click()
```

Comment:
Hm it looks like this method is used by other UI tests so those will likely break now? (Haven't had a chance to test on device yet)

Reply:
Sorry I missed that. Since we don't need the close button any more, I just removed that test in the latest commit.